otx/algorithms/detection/adapters/mmdet/task.py

```diff
@@ -626,8 +633,7 @@ def _init_deploy_cfg(self, cfg) -> Union[Config, None]:
             deploy_cfg = MPAConfig.fromfile(deploy_cfg_path)
 
             patch_input_preprocessing(cfg, deploy_cfg)
-            if not deploy_cfg.backend_config.get("model_inputs", []):
-                patch_input_shape(cfg, deploy_cfg)
+            patch_input_shape(cfg, deploy_cfg)
```

Comment:
@sungmanc Shouldn't we revert this change since this PR remove backend_config of mask_rcnn's deployment.py

Reply:
Not mandatory, however make sense.

Reply:
`patch_input_shape()` sets input shape for export from test pipeline info. Do we need any condition?

Reply:
It seems that patch_input_shape directly assign new model_inputs to deploy_cfg.backend_config. Then I think it is okay to leave as it is